<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extrinsic</class>
 <widget class="QDialog" name="extrinsic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extrinsic calibration</string>
  </property>
  <widget class="QSpinBox" name="numPicsDropdown">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectPair">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select camera pair</string>
   </property>
  </widget>
  <widget class="QLabel" name="photoCount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Photo count</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrateButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="calibResult">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration result:</string>
   </property>
  </widget>
  <widget class="QLabel" name="calibResultText">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
